static/sass/_pattern_p-strip.scss

```diff
@@ -4,6 +4,35 @@
   @include jaas-p-strip-half-and-half;
   @include jaas-p-strip-muted-suru;
   @include charmhub-p-strip;
+  @include p-suru-light;
+}
+
+@mixin p-suru-light {
+  .has-suru {
```

Comment:
Sorry, but its probably worth scoping this to just strips using `[class*='p-strip']` for example https://github.com/canonical-web-and-design/vanilla-framework/blob/master/scss/_patterns_strip.scss#L103.